model/config.go

```diff
@@ -513,11 +513,6 @@ func (o *Config) SetDefaults() {
 		o.FileSettings.AmazonS3Endpoint = "s3.amazonaws.com"
 	}
 
-	if o.FileSettings.AmazonS3Region == "" {
```

Comment:
Actually @hmhealey default region never really worked, in-fact Region setting had no affect whatsoever since probably last minio-go refactor.